### PR TITLE
fix: commit limit when rebasing

### LINKED ISSUE #6622: prCommitsPerRunLimit is ignored when rebaseWhen:"behind-base-branch" is set
**What Renovate type are you using?**

Renovate cli on Azure

**Describe the bug**

When running the cli with `prCommitsPerRunLimit` set to 1 and `rebaseWhen:"behind-base-branch"` renovate will update all pr's with latest changes from master instead of only updating 1 pr that run.

**To Reproduce**

* reproduction repo has 2 branches created before the readme.md had the text "update to Readme" added to master
* run renovate cli with prCommitsPerRunLimit set to 1 on the repo at https://github.com/tony1377/renovate-test-4/
* expected - renovate only updates 1 pr with the updates from master
* actual - renovate updates both pr's with the updates from master

### PR DESCRIPTION
**Pull request by rarkins**
Closes #6622

### CODE REVIEW

File: lib/workers/branch/index.ts [hunk 1]
```diff
@@ -56,7 +56,8 @@ const rebasingRegex = /\*\*Rebasing\*\*: .*/;
 
 export async function processBranch(
   branchConfig: BranchConfig,
-  prHourlyLimitReached?: boolean
```
Comment (rarkins) [inline lib/workers/branch/index.ts:59]: This name was wrong as it also enforces concurrent

File: lib/workers/branch/index.ts [hunk 2]
```diff
@@ -148,14 +149,20 @@ export async function processBranch(
     }
     if (
       !branchExists &&
```
Comment (rarkins) [inline lib/workers/branch/index.ts:151]: If branch exists then doesn't apply to PR limits as they are enforced on creation

File: lib/workers/repository/dependency-dashboard.ts [hunk 3]
```diff
@@ -96,7 +96,8 @@ export async function ensureMasterIssue(
     issueBody += '\n';
   }
   const rateLimited = branches.filter(
-    (branch) => branch.res && branch.res.endsWith('pr-hourly-limit-reached')
+    (branch) =>
+      branch.res === 'pr-limit-reached' || branch.res === 'commit-limit-reached'
```
Comment (viceice) [inline lib/workers/repository/dependency-dashboard.ts:100]: Better to use constants / enums for this.

**Review by viceice — COMMENTED** [review]
I think those strings should be constants or enums (can be done in seperate pr). Otherwise LGTM.

**Pull request merged**